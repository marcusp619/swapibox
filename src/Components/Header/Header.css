/*------------------------------------*\
  #HEADER
\*------------------------------------*/
.Header {
  background: peru;
  display: flex;
  justify-content: space-between;
}
