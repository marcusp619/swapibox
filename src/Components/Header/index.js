import React, { Component } from 'react';
import './Header.css';

class Header extends Component {
  constructor(props) {
    super(props)
  }

  render() {
    return(
    <div className="Header">
      <h2>Header Component</h2>
    </div>
    )
  }
}

export default Header;