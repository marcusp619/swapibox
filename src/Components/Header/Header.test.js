import React from 'react';
import Header from '.';
import { shallow } from 'enzyme';

describe('Header', () => {
it('renders without crashing', () => {
  const wrapper = shallow(<Header />);

   expect(wrapper).toMatchSnapshot();
  });
}); 