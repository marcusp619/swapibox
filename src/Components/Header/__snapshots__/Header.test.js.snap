// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<div
  className="Header"
>
  <h2>
    Header Component
  </h2>
</div>
`;
